pkg/backends/jmeter/resources.go

```diff
@@ -56,7 +54,7 @@ var (
 )
 
 // NewJMeterSettingsConfigMap creates a new configmap which holds jmeter config
-func (c *JMeter) NewJMeterSettingsConfigMap() *coreV1.ConfigMap {
+func (b *Backend) NewJMeterSettingsConfigMap() *coreV1.ConfigMap {
```

Comment:
Why was this changed? This breaks BC at the moment right? 
(Same question for other `func (c *JMeter)` to `func (b *Backend) ` changes.)

Reply:
`JMeter` now only have aliases to `Backend` functions, so this needs to be moved to new `Backend` struct as well.
Once the switch to this new registry pattern is done, all this `JMeter` struct can be deleted.

Reply:
I know but right now the tests no longer cover `JMeter` meaning people can break tests easily for `JMeter` but keep `Backend` tests passing. 
This in turn leads to other issues. I would advice to move the tests once you remove `JMeter` and we switch to using `Backend` directly

Reply:
That's the plan, but I also had to adapt the test, I will tag you on the related code.